### DISCUSSION TITLE
Use tag instead of manually transude dashboard

### DISCUSSION THREAD

**Discussion by kookma** [NicolasPetton/Projectify#4] [Q&A] [answered]
In Usage it stated that

> ## How to display the project dashboard
> The main entry point to the Project Manager plugin is the dashboard. The dashboard can be embedded (transcluded) in any tiddler using the following:
> 
>` {{$:/plugins/nico/pm/ui/dashboard/Dashboard}}`

While this is quite acceptable and semantic but you can have a ViewTemplate like below

```
<$list filter="[all[current]tag[Dashboard]]">
{{$:/plugins/nico/pm/ui/dashboard/Dashboard}}
</$list>
```

Now, any tiddler tagged with **Dashboard** it will embed the dashboard in itself.

- Drag and drop attached tiddler into Project manager wiki
- Create a new tiddler
- Tag it with Dashboard
- Save

[dashboard_viewtemplate.zip](https://github.com/NicolasPetton/ProjectManager/files/5683151/dashboard_viewtemplate.zip)

**Comment by NicolasPetton** [comment, marked as answer]
Projectify is using ViewTemplates for todos and projects.

In this case however I'm not sure it would be better than transcluding the dashboard tiddler, since there can only be one dashboard.

**Reply by kookma** [reply]
Yes, having only one Dashboard makes sense to use direct transclusion as it seems to be faster than ViewTemplate.
